.github/workflows/wasm-benhmarks.yml

```diff
@@ -0,0 +1,99 @@
+name: Wasm perf
+on:
+  pull_request:
```

Comment:
As it stands, it looks like this workflow is applied to every PR. In your opinion, would it make sense to only run it to a subset of the PRs, e.g. the ones that explicitly contain "wasm" or "napi" in their branch name?

Alternatively: should we restrict this workflow to run only on changes affecting `quaint`, `driver-adapters`, `query-engine-wasm`, or `query-engine-node-api`?

Reply:
Good observation! 

I think we should run it for each brach, given that any change in the code that could affect transitively the code in the query engine wasm module can affect as well the benchmark results.  As for the exclusions, I need to take back a similar pattern than that of the wasm-size workflow. (I removed it while I was making changes to see the results no matter what I changed) as soon as this is stabilized and approved I will take them back.

Reply:
Nice, I'm totally cool with these refinement being applied in a follow-up PR